tests/light/functional_tests/filterx/test_filterx_cache_json_file.py

```diff
@@ -45,3 +47,10 @@ def test_filterx_cache_json_file_reloads_its_content_automatically(syslog_ng, co
         file.write('{"msg": "autoupdated"}')
 
     assert destination.read_until_logs(["autoupdated"])
+
```

Comment:
I'd prefer this to a be a separate testcase if possible.

Reply:
updated